### PR TITLE
princejacobkuruvilla's caption

### LINKED ISSUE #1: Comment here to be added as a collaborator
### Comment below - you can say anything 
 We will all be using this repository today. Even though it's public right now, you will not be able to make changes until you're given the correct permissions. We've automated this process via the GitHub API and some fancy chatops. Once you comment, we will add you as a collaborator. 

  You'll also start to receive a lot of emails. (:exclamation:) You can slow these down immediately by clicking the 'Unwatch' button and selecting either the 'Not watching' or 'Ignoring' option.

### PR DESCRIPTION
**Pull request by princejacobkuruvilla**
based on class instructions made changes to the caption. additional details goes here!

fixes #1

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
doing a review

File: _posts/0000-01-07-princejacobkuruvilla.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+One flew over the cuckoo's nest
```
Comment (ghost) [inline _posts/0000-01-07-princejacobkuruvilla.md:6]: review comment directly in the code

File: _posts/0000-01-07-princejacobkuruvilla.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (ghost) [inline _posts/0000-01-07-princejacobkuruvilla.md]:
```suggestion
title: "Welcome to our slide deck!" suggest new title
```

**Review by ghost — DISMISSED** [review]
make changes as needed

**Review by beardofedu — APPROVED** [review]

**Pull request merged**